src/validate_pyproject/plugins/__init__.py

```diff
@@ -20,12 +20,9 @@
     from importlib_metadata import EntryPoint, entry_points
```

Comment:
This can be fixed. Ruff's PYUP code should fix this automatically, I think, do we not have that on because it didn't support 3.6?

Reply:
(I could only comment on the last line of the block, but pretend it's over the whole block)

Reply:
Thank you very much Henry for double checking this.

I don't remember if `ruff` is installed in the CI for this repo right now (my main grudge with tools implemented in Rust is the lack of support for some non-mainstream but conceptually very important OSs like OpenBSD), but I did run it manually...

I did try running the following:

```bash
ruff check src/validate_pyproject --select=UP --fix --preview --unsafe-fixes
cd src
ruff check validate_pyproject --select=UP --fix --preview --unsafe-fixes
pipx run --python python3.12 ruff check validate_pyproject --select UP --fix --preview --unsafe-fixes
cd ..
pipx run pyupgrade --py38-plus src/**/*.py tests/**/*.py
pipx run --python python3.12 ruff format src
21 files left unchanged
```

No extra changes suggested by the tool.

(`ruff check --select ...` based on https://docs.astral.sh/ruff/formatter/#sorting-imports)

I will fix the issues manually.

Reply:
It shows up for me if I change:

https://github.com/abravalheri/validate-pyproject/blob/3efd7d1f275c259c44ec3f47b13d9e4ead14e6e1/.ruff.toml#L3

Reply:
I see. Forgot that, thank you Henry.